k8s-plugin/src/main/kotlin/com/amazon/spinnaker/keel/k8s/K8sConfig.kt

```diff
@@ -15,7 +15,9 @@ const val K8S_PROVIDER = "kubernetes"
 const val SOURCE_TYPE = "text"
 
 const val K8S_LAST_APPLIED_CONFIG: String = "kubectl.kubernetes.io/last-applied-configuration"
-const val KEEL_MONIKER_APP: String = "moniker.spinnaker.io/application"
+const val KEEL_MONIKER_APP_ANNOTATION: String = "moniker.spinnaker.io/application"
+const val MANAGED_DELIVERY_APP_LABEL = "md.spinnaker.io/plugin"
+const val MANAGED_DELIVERY_K8S_PLUGIN = "k8s"
```

Comment:
Wonder if we should create a list of label pairs to apply instead since they are tightly coupled. We may need to add more labels in the future as well?
 `val MANAGED_DELIVERY_APP_LABEL = listOf(Pair("md.spinnaker.io/plugin", "k8s"))`

Reply:
yeah good idea! will fix.